csuntechlab/iSTART
# STR-49 -- FE: Verify a users session inactivity

# Description
'Automatically logs out user after 27 minutes of inactivity per CSUN's standard'
  
# Testing
1. add the .env variable i sent in my email to your .env file
2. change the variable's value to something like .125 (.125 = ~7 seconds after conversion)
2. login
3. interact with the page in some small way: move the mouse around, click, etc
3. Do nothing in the window: do not move mouse, touch keyboard, scroll wheel, etc.
4. wait
5. You should be logged out after the time specified in the env value
